compiler/rustc_lint/src/unused.rs

```diff
@@ -805,6 +805,18 @@ impl EarlyLintPass for UnusedParens {
                 }
                 return;
             }
+            ExprKind::Match(ref _expr, ref arm) => {
+                for a in arm {
+                    self.check_unused_delims_expr(
+                        cx,
+                        &a.body,
+                        UnusedDelimsCtx::BlockRetValue,
+                        true,
```

Comment:
```suggestion
                        false,
```
I don't think match arms are followed by a block?